config/config.proto

```diff
@@ -60,14 +75,10 @@ message JobConfig {
 	optional string scrape_interval = 2;
 	// Per-target timeout when scraping this job. Must be a valid Prometheus
 	// duration string in the form "[0-9]+[smhdwy]".
-	optional string scrape_timeout = 7 [default = "10s"];
-	// The DNS-SD service name pointing to SRV records containing endpoint
-	// information for a job. When this field is provided, no target_group
-	// elements may be set.
-	optional string sd_name = 3;
-	// Discovery refresh period when using DNS-SD to discover targets. Must be a
-	// valid Prometheus duration string in the form "[0-9]+[smhdwy]".
-	optional string sd_refresh_interval = 4 [default = "30s"];
+	optional string scrape_timeout = 3 [default = "10s"];
+	// The service discovery information for a job.
```

Comment:
I'd like to be able to get multiple jobs from one block, so when I add a new job I don't have to make config changes.

Should this field be repeated?